Nautilus/Assets/Gadgets/CraftingGadget.cs

```diff
@@ -103,6 +100,11 @@ protected internal override void Build()
             {
                 CraftTreeHandler.AddCraftingNode(FabricatorType, prefab.Info.TechType, StepsToFabricatorTab);
             }
+            
+        }
+        else if (!prefab.TryGetGadget(out ScanningGadget scanningGadget) || !scanningGadget.IsBuildable)
+        {
+            InternalLogger.Log($"Prefab '{prefab.Info.ClassID}' was not automatically registered into a crafting tree.");
```

Comment:
Think we could make this Debug log instead?

Reply:
I think marking that as a debug log will make it easy to miss because a craftable item without a crafting tree that isn't a buildable is alarming and it's safer to assume that the developer missed something. In fact, I'd even argue it should be a warning instead.